client/retry/backoff.go

```diff
@@ -16,12 +16,27 @@ package retry
 
 import (
 	"context"
+	"reflect"
+	"runtime"
+	"strings"
 	"time"
 
 	"github.com/pingcap/errors"
 	"github.com/pingcap/failpoint"
+	"github.com/pingcap/log"
+	"go.uber.org/zap"
 )
 
+// Option is used to customize the backoffer.
+type Option func(*Backoffer)
+
+// WithLogTimes sets the number of retries required to print a log.
+func WithLogTimes(logTimes int) Option {
```

Comment:
I think it should belong to the log component?

Reply:
But the log component doesn't know how many times the backoffer has done?
How about providing a function param that the caller defines? like func `WithActionAfterTimes(times int, fn func()) Option`

Reply:
For the caller, I think it's different to know how to set this value, it's affected by the backoff policy. If we must do it, I prefer something like withMinLogInterval.